.meepctl-repocfg.yaml

```diff
@@ -42,6 +44,25 @@ repo:
       ca: self-signed
       # lets-encrypt production server (true) or staging server (false)
       le-server-prod: false
+      
+    auth:
+      # session encryption key
+      session-key: my-secret-key
```

Comment:
This should point to a K8s secret name which holds the secret key

Reply:
Agreed. This will be addressed in an upcoming PR.